static/scenes/railroad/js/camera-system.js

```diff
@@ -1,16 +1,21 @@
-export class CameraSystem {
+goog.provide('app.CameraSystem');
+
+class CameraSystem {
 
   constructor(camera, placeholderScene) {
     this.camera = camera;
     this.placeholderScene = placeholderScene;
   }
 
-  update() {
-    // TODO: Once we have more game logic in, this shouldn't directly use the
-    // real world time but probably update based on some delta time.
-    const nowSeconds = Date.now() / 1000;
-    this.camera.position.copy(this.placeholderScene.getCameraPosition(nowSeconds));
+  update(deltaSeconds) {
+    if (!this.seconds) {
+      this.seconds = Date.now() / 1000;
```

Comment:
Initialize this to zero, so we always start at the same place.

Reply:
Done, thanks for the tip!